packages/flutter_tools/lib/src/base/os.dart

```diff
@@ -184,20 +184,22 @@ class _PosixUtils extends OperatingSystemUtils {
 
   @override
   void chmod(FileSystemEntity entity, String mode) {
+    // Errors here are silently ignored (except when tracing).
     try {
       final ProcessResult result = _processManager.runSync(
         <String>['chmod', mode, entity.path],
       );
       if (result.exitCode != 0) {
         _logger.printTrace(
-          'Error trying to run chmod on ${entity.absolute.path}'
-          '\nstdout: ${result.stdout}'
-          '\nstderr: ${result.stderr}',
+          'Error trying to run "chmod $mode ${entity.path}":\n'
```

Comment:
why not absolute path?

Reply:
I just figured it would be less confusing if the error message exactly matched the actual call we were making. I don't feel strongly about it though if you would rather we keep it absolute.